cristin-import/src/test/java/no/unit/nva/cristin/mapper/CristinMapperTest.java

```diff
@@ -233,6 +243,75 @@ public void mapReturnsPublicationWithHardcodedLink() {
         assertThat(actualLicenseIdentifiers, is(equalTo(expectedLinks)));
     }
 
+    @ParameterizedTest(name = "map returns NVA role {1} when Cristin role is {0}")
+    @CsvSource({"REDAKTØR,EDITOR", "FORFATTER,CREATOR"})
+    public void mapReturnsPublicationsWhereCristinEditorRoleIsMappedToCorrectNvaEditorRole(String cristinRole,
+                                                                                           String nvaRole) {
+        CristinContributorRoleCode actualCristinRole = CristinContributorRoleCode.fromString(cristinRole);
+        Role expectedNvaRole = Role.lookup(nvaRole);
+        CristinObject objectWithEditor =
+            cristinObjects().filter(
+                cristinObject -> cristinObjectHasRole(cristinObject, actualCristinRole))
+                .findFirst()
+                .orElseThrow();
+
+        Optional<Contributor> contributor = objectWithEditor.toPublication()
+                                                .getEntityDescription()
+                                                .getContributors()
+                                                .stream()
+                                                .filter(c -> c.getRole().equals(expectedNvaRole))
+                                                .findAny();
+        assertThat(contributor.isPresent(), is(true));
+        assertThat(contributor.orElseThrow().getRole(), is(equalTo(expectedNvaRole)));
+    }
+
+    @Test
+    public void mapReturnsPublicationsWhereCristinCreatorRoleIsMappedToNvaCreatorRole() {
```

Comment:
Is this name the right one? you cast an exception and assert that, you do not assert that you have a valid map of publications with their roles? or am i wrong?

Reply:
it was the wrong test name. In general, I am taking the approach that if I do not know the mapping of a Crstin value to an NVA value, it is an error. The mapping will be finished when there are no errors left